tests/core/test_model.py

```diff
@@ -1969,16 +1969,36 @@ def test_custom_interval_unit():
         )
 
 
-def test_model_table_properties(sushi_context):
-    # # Validate python model table properties
-    assert sushi_context.models["sushi.items"].table_properties == {
+def test_model_table_properties() -> None:
+    # Validate python model table properties
+    @model(
+        "my_model",
+        kind="full",
+        columns={"id": "int"},
+        table_properties={
+            "format": "PARQUET",
+            "bucket_count": 0,
+            "orc_bloom_filter_fpp": 0.05,
+            "auto_purge": False,
+        },
+    )
+    def my_model(context, **kwargs):
+        pass
+
+    m = model.get_registry()["my_model"].model(
```

Comment:
[Nit] should we rename this to `python_model = ...` for consistency with the `sql_model` changes below?